src/components/CustomerQuota/ItemsSelection/ItemContent.tsx

```diff
@@ -28,13 +29,17 @@ export const ItemContent: FunctionComponent<{
   unit: CampaignPolicy["quantity"]["unit"];
   maxQuantity: number;
   accessibilityLabel?: string;
+  showAddonsToggle: (e: GestureResponderEvent) => void;
+  showAddons: boolean;
```

Comment:
These should be optional. `ItemContent` should not always need to have the concept of "addons"